code/modules/food_and_drinks/kitchen_machinery/gibber.dm

```diff
@@ -152,10 +152,10 @@
 	var/mob/living/mob_occupant = occupant
 	var/sourcename = mob_occupant.real_name
 	var/sourcejob
-	if(ishuman(occupant))
-		var/mob/living/carbon/human/gibee = occupant
-		sourcejob = gibee.job
-	var/sourcenutriment = mob_occupant.nutrition / 15
+	var/sourcenutriment
```

Comment:
should probably be `0` by default? or maybe just cancel out if the occupant is not living?